lib/autoyast.rb

```diff
@@ -23,6 +23,15 @@ def initialize(description)
       "repositories",
       "packages"
     )
+    if !description.users
+      Machinery::Ui.puts(
+        "\nWarning: Exporting a description without the scope 'users' as AutoYaST" \
+        " profile will result in a root account without a password which prevents" \
+        " logging in.\n" \
+        "So either inspect the scope 'users' before the export or" \
```

Comment:
For now `machinery inspect` is the most common way to get a system description. But, there will be other ways to assemble a system description in the long run, e.g. using an editor. For now the current text looks okay, if we want to have the text future proof, I'd recommend: "So either add a 'users' section to the system description (e.g. by inspecting scope users) or"